pkg/chain_iterator/block_batch_iterator.go

```diff
@@ -19,6 +19,7 @@ import (
 const (
 	DefaultBlocksReadPerEpoch = 1000
 	DefaultRetryInterval      = 12 * time.Second
+	DefaultBlockConfirmations = 0
```

Comment:
If BlockConfirmations is not set default value is 0, so this field is not necessary.

Reply:
ok